rj_gameplay/rj_gameplay/play/basic_defense.py

```diff
@@ -36,8 +34,14 @@ def tick(
 
         if self._state == State.INIT:
             self.prioritized_tactics.append(goalie_tactic.GoalieTactic(world_state, 0))
-            self.prioritized_tactics.append(wall_tactic.WallTactic(world_state, 3))
-            self.prioritized_tactics.append(nmark_tactic.NMarkTactic(world_state, 2))
+            num_wallers = min(3, len(world_state.our_visible_robots) - 1)
+            self.prioritized_tactics.append(
+                wall_tactic.WallTactic(world_state, num_wallers)
```

Comment:
We still need to change the position of the line for this play, but that would be in a different PR.